dev/VSIX/Extension/ProjectReunion.Extension.csproj

```diff
@@ -56,11 +56,11 @@
     <Reference Include="System" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.VisualStudio.SDK" Version="16.0.206" ExcludeAssets="runtime">
+    <PackageReference Include="Microsoft.VisualStudio.SDK" Version="17.0.0-previews-1-31325-273" ExcludeAssets="runtime">
```

Comment:
Can you open a bug to track updating this to release version when available?

Reply:
Done.